### DISCUSSION TITLE
Error 'Breakpoint A breakpoint has been reached.' occurred while running cefsharp.browsersubprocess

### DISCUSSION THREAD

**Discussion by Nachtinho** [cefsharp/CefSharp#4994] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

130.1.90.0

### Operating System

Windows 11

### Architecture

x64

### .Net Version

.NET 4.8

### Implementation

OffScreen

### Reproduction Steps

Recently, a problem has arisen with the Cefsharp browser (see title).

We start the browser through a C# process that launches a Windows console application in which the Cefsharp browser is implemented.

When I run the calling process as an administrator, it works.
When I directly invoke the console application, it also works.
However, when I start the process as a normal user, an error message appears in a dialog, and the process aborts.

I have created a debug.log with following entries:
[32080:28904:1127/090839.916:VERBOSE1:partitioned_lock_manager.cc(187)] All locks acquired for Start@chrome\browser\web_applications\generated_icon_fix_manager.cc:58
[32080:28904:1127/090839.916:VERBOSE1:partitioned_lock_manager.cc(229)] Releasing <PartitionedLockId>{id: 0x31, partition: 0} requested by Start@chrome\browser\web_applications\generated_icon_fix_manager.cc:58
[32080:28904:1127/090839.934:FATAL:check.cc(361)] Check failed: false. NOTREACHED log messages are omitted in official builds. Sorry!

while running as an administrator, I found the same entries but the last one. 
Instead it continues with this:
[21324:33072:1127/092141.745:VERBOSE1:frame_host_impl.cc(573)] frame 5-32EC81229C8F4130E3DCC5A6453C4971 (main) detached (reason=RENDER_FRAME_CHANGED, is_connected=0)

### Expected behavior

Should not throw this error

### Actual behavior

throws exception in a dialog and stops

### Regression?

It happens with older versions, too

### Known Workarounds

works as administrator

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Converting to discussion as this is unlikely a bug as two out of three run successfully.


> When I directly invoke the console application, it also works.
> However, when I start the process as a normal user, an error message appears in a dialog, and the process aborts.

What's the difference between the two? You'll need to do some investigation. Working directory? etc

**Comment by tux1337** [comment]
@amaitland 
We have a similar issue as described and can reproduce the issue with Version 135.0.170; it works definitely with Version 120.2.70.
May you guide us how to debug this, please?

**Reply by amaitland** [reply]
> What's the difference between the two? You'll need to do some investigation. Working directory? etc

@tux1337 Same as above

Start with the simple things.



```c#
string exeDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
Directory.SetCurrentDirectory(exeDir);
```

**Comment by pipiscrew** [comment]
getting the same error on **win10x64** with **nuget** dlls for **v145.0.260x64** with **.net framework v4.6.2**

**files referenced** 

- cefsharp.winforms.145.0.260.nupkg
- cefsharp.common.145.0.260.nupkg
- chromiumembeddedframework.runtime.win-x64.145.0.26.nupkg

on the file **CefSharp.Core.Runtime.dll**

on the below **function** getting the error (after debugging)
`public unsafe static bool **Initialize**(CefSettingsBase cefSettings, [MarshalAs(UnmanagedType.U1)] bool performDependencyCheck, IApp cefApp)`

on this **line**
`flag = (<Module>.CefInitialize(ref cefMainArgs, ref cefStructBase<CefSettingsTraits>, (scoped_refptr<CefApp>*)(&scoped_refptr<CefApp>), null) != null);`

## CAUSE : 
needs the 
`\locales`
folder with the **files**.. ;)  which exist to chromiumembeddedframework.runtime.win-x64.145.0.26.nupkg

please close the issue!
